helm/_shared/named_templates/_postgres.tpl

```diff
@@ -60,3 +60,34 @@
       name: vro-db
       key: DB_CLIENTUSER_PASS
 {{- end }}
+
+{{- define "domainCc.alembic.envVars" -}}
```

Comment:
something beyond my helm chart knowledge, not sure it's noteworthy: there's another partial template using this name: https://github.com/department-of-veterans-affairs/abd-vro/blob/7f3468fb67cd5e9bb7e56818f165a3852fa43928/helm/domain-cc/templates/_alembicdb.tpl#L4

Reply:
you are right about this.  helm won't have a problem with it, because it is good at overlaying templates like this, but it does get confusing for the humans trying to follow what happens. What we really want is for the domainCc.alembic.envVars to contain the flyway variables for the db-init service and for domain-cc to contain the plain DOMAIN_CC variables.  The current setup accomplishes that but it isn't clear why and we could clean that up a bit.